app/Http/Client/Alchemy/AlchemyPendingRequest.php

```diff
@@ -321,6 +321,13 @@ public function parseNft(array $nft, int $networkId): Web3NftData
 
         $mintTimestamp = $this->getNftMintingDateProperty($nft);
 
+        $bannerImageUrl = Arr::get($nft, 'contractMetadata.openSea.bannerImageUrl');
+        if (! empty($bannerImageUrl)) {
+            $bannerImageUrl = preg_replace('/(?<=\?|&)w=(\d+)/', 'w=1378', $bannerImageUrl);
```

Comment:
why do the replacement here when there is one handling it in `NftImageUrl`? 🤔 I'd prefer if that one gets updated to match the available options from OpenSea so it's all in one place and that passes a size as well as opposed to changing it to a hardcoded value